web/js/containers/share.js

```diff
@@ -243,15 +246,15 @@ class ShareLinkContainer extends Component {
               Copy URL to share link.
             </p>
             {' '}
-            {urlShortening && (
+            {urlShortening && showShortenUrlCbx ? (
```

Comment:
I think you should be able to omit the tertiary here and just check for 
```{urlShortening && showShortenUrlCbx && (...)}```

Reply:
Agreed & addressed in b583bc219db4798d42179f84898a0be041efc8f1.